app/models/dependent.rb

```diff
@@ -170,11 +170,11 @@ def eligible_for_eip2?
   end
 
   def qualifying_child_relationship?
-    QUALIFYING_CHILD_RELATIONSHIPS.include? relationship
+    QUALIFYING_CHILD_RELATIONSHIPS.include? relationship.downcase
```

Comment:
Would this be tidier as a validation on `relationship`? Why would we ever get non-downcase'd relationships, and are non-downcase'd relationships stored in the database? Can we avoid that?

Not a blocker.

Reply:
no we can't avoid it because we use the same field to store free-form relationships on GYR intake and input from forms, etc. It's possible to create a conditional validation based on the type of intake the dependent is associated with, but that can still "silently" disqualify dependents in tests if they're associated to an intake after creation, or cause issues with updating existing dependents who have "invalid" dependent relationships based on newer logic

I think we should "fix" this by making relationship an enum field and remove the free form entry for dependents on GYR intake.